### ISSUE TITLE
BIC/AIC/DOF not correct since Version 0.85

### ISSUE THREAD

**Issue by 3fon3fonov** [3fon3fonov/exostriker#125]
I suspect that rvmod_for.f95 returns a wrong 'mfit' output?

TBCheked!

**Comment by 3fon3fonov** [comment]
Sorry I will have to take that back..... my mistake from a previous fit. However, BIC/AIC and DOF seems to not update properly from the GUI side when the fit is initialized with a different number of fitted parameters.....  More investigation is needed!

**Comment by Astrocyy** [comment]
Hi Trifon,

When I tried to install the latest version of Exostriker within the anaconda environment, one of the errors is about "rv_mod", here is the error shown in the terminal:

`ImportError: cannot import name 'rvmod_for' from partially initialized module 'RV_mod.rvmod' (most likely due to a circular import)`

Now I just passed using Anaconda and it becomes super easy and smoothly to install the Exostriker(0.86 ver.)! So, the issue about "rv_mod" won't be a problem anymore...

But everytime I open the exostriker, this warning occurs: 
(I don't know if it's the problem of the Python not about Exostriker and I have no idea how to fix it or just neglect it...)
```
0.00s - Debugger warning: It seems that frozen modules are being used, which may
0.00s - make the debugger miss breakpoints. Please pass -Xfrozen_modules=off
0.00s - to python to disable frozen modules.
0.00s - Note: Debugging will proceed. Set PYDEVD_DISABLE_FILE_VALIDATION=1 to disable this validation.
qt.qpa.fonts: Populating font family aliases took 145 ms. Replace uses of missing font family "Times" with one that exists to avoid this cost. 
```

However, I found that it became harder to make a better fit. I mean, it could derive a quite correct and good result by merely click the "RV Auto fit" in the old ver. of Exostriker, but now, it's pretty difficult to fit it well with the same dataset. I don't know if this is a problem...

So, these are a few problems after installing the latest version of Exostriker on iMac(Mac Ventura 13.5.2, intel core i7).

Looking forward to your reply! Thanks!

Best,
Yingyi
